### PR TITLE
Market data freiburg

### PR DESCRIPTION
**Pull request by rackedbrain**
added market data for the city of freiburg

### CODE REVIEW

**Comment by johnjohndoe** [comment]
@rackedbrain Do you need help to sort out the errors?

**Comment by rackedbrain** [comment]
Yes, that would be nice!

Is it an encoding problem?

**Comment by johnjohndoe** [comment]
@rackedbrain Not sure yet. But even the preview of the GeoJSON file as provided by GitHub does not work, see [here](https://github.com/rackedbrain/wo-ist-markt.github.io/blob/market-data-freiburg/cities/freiburg.json).

**Comment by k-nut** [comment]
It looks from here as if it is the quotes. Stating in lines [18](https://github.com/rackedbrain/wo-ist-markt.github.io/blob/market-data-freiburg/cities/freiburg.json?short_path=db453eb#L18) you are sometimes using German quotes (`”`) instead of the standard ascii ones (`"`). Changing that should help ☀️

**Comment by johnjohndoe** [comment]
@rackedbrain Did you see you can inspect the build errors yourself? Just click the link above pointing to Travis CI. The console output there will point you to the possible error in the file.

**Comment by rackedbrain** [comment]
@johnjohndoe Yes, I'm already checking...

File: cities/freiburg.json [hunk 1]
```diff
@@ -189,8 +189,7 @@
                 "type": "Point"
             },
             "properties": {
-                "location": "Stühlinger Kirchplatz
-unter Stadtbahnbrücke",
+                "location": "Stühlinger Kirchplatz",
```
Comment (johnjohndoe) [inline cities/freiburg.json]: I think, you could just put it in one line.

**Comment by rackedbrain** [comment]
Now it says 
`undefined:265`
`],`
My editor says, this bracket matches one in line 8, so I don't know what is wrong here...

File: cities/freiburg.json [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Platz der Zähringer",
+                "opening_hours": "We, Sa 08:00-13:00",
+                "title": "Zähringer Bauernmarkt"
+            },
+            "type": "Feature"
+        },
```
Comment (johnjohndoe) [inline cities/freiburg.json]: Remove the `,` after the last `}` of the last feature block.

File: cities/freiburg.json [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+            "geometry": {
+                "coordinates": [
+                    7.85344,
+                    47.98480
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Alter Wiehrebahnhof",
+                "opening_hours": "We 14:00-18:30: Sa 08:00-13:00",
```
Comment (johnjohndoe) [inline cities/freiburg.json]: Fix the opening hours `We 14:00-18:30: Sa 08:00-13:00`

File: cities/freiburg.json (hunk 2 above)
Comment (rackedbrain) [inline cities/freiburg.json]: Seems to work, thank you!

**Comment by johnjohndoe** [comment]
@rackedbrain Can you please put all your changes in a **single commit**? The following commands should do it:

``` bash
$ git reset cd43f3 // The root commit of your branch
$ git add .
$ git commit -m "Add market data for Freiburg."
$ git push -f origin market-data-freiburg
```

**Comment by johnjohndoe** [comment]
@rackedbrain Nice. Thanks for adding Freiburg. I am looking forward to reading a **tweet** from you :} Please add a screenshot and the URL. Further, it would be great if you could convince the city to link to this website. See #38.

**Pull request closed without merging**